src/tests/unit/tests/electron/views/root-container/components/root-container.test.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+
+import { BaseClientStoresHub } from 'common/stores/base-client-stores-hub';
+import { ClientStoresHub } from 'common/stores/client-stores-hub';
+import { WindowStateStore } from 'electron/flux/store/window-state-store';
+import { WindowStateStoreData } from 'electron/flux/types/window-state-store-data';
+import { DeviceConnectViewContainerState } from 'electron/views/device-connect-view/components/device-connect-view-container';
+import {
+    RootContainer,
+    RootContainerDeps,
+    RootContainerProps,
+    RootContainerState,
+} from 'electron/views/root-container/components/root-container';
+import { shallow } from 'enzyme';
+import { It, Mock } from 'typemoq';
+
+describe(RootContainer, () => {
+    let storeListener: () => Promise<void>;
+    let deps: RootContainerDeps;
+    let windowStateStoreData: WindowStateStoreData;
+
+    beforeEach(() => {
+        windowStateStoreData = {
+            routeId: 'deviceConnectView',
+        };
+
+        const storeHubMock = Mock.ofType<
+            ClientStoresHub<DeviceConnectViewContainerState> & ClientStoresHub<WindowStateStore> & ClientStoresHub<RootContainerState>
```

Comment:
is ` ClientStoresHub<WindowStateStore>` really needed here?

Reply:
it is not and I have removed it in this next commit